Source/JavaScriptCore/wasm/WasmAirIRGenerator.cpp

```diff
@@ -3386,6 +3397,104 @@ auto AirIRGenerator::addArrayLen(ExpressionType arrayref, ExpressionType& result
     return { };
 }
 
+auto AirIRGenerator::addStructNew(uint32_t typeIndex, Vector<ExpressionType>& args, ExpressionType& result) -> PartialResult
+{
+    ASSERT(typeIndex < m_info.typeCount());
+    result = tmpForType(Type { TypeKind::Ref, m_info.typeSignatures[typeIndex]->index() });
+    auto jsInstance = g64();
+    append(Move, Arg::addr(instanceValue(), Instance::offsetOfOwner()), jsInstance);
+    // FIXME: inline the allocation.
+    // https://bugs.webkit.org/show_bug.cgi?id=244388
+    emitCCall(&JSWebAssemblyStruct::tryCreate, result, jsInstance, addConstant(Types::I32, typeIndex));
```

Comment:
You cannot call a function which is not defined with `JSC_DECLARE_JIT_OPERATION`. So you need to use operationWasmStructNew, or define a function in WasmOperations.cpp.